### PR TITLE
add UT with vitest:ui

### PR DESCRIPTION
**Pull request by guoxianzhe**
chore:add patches to fix vitest types error

THIS IS IN PROGRESS

### CODE REVIEW

File: packages/agora-rtc-react/src/components/CameraVideoTrack.tsx [hunk 1]
```diff
@@ -30,7 +30,7 @@ export function CameraVideoTrack({ track: maybeTrack, deviceId, ...props }: Came
 
   useEffect(() => {
     if (track && deviceId != null) {
-      track.setDevice(deviceId).catch(console.warn);
+      track.setDevice(deviceId);
```
Comment (LichKing-2234) [inline packages/agora-rtc-react/src/components/CameraVideoTrack.tsx:33]: 这里的catch去掉的原因是什么

**Comment by LichKing-2234** [comment]
需要添加UT对应的Action check

**Review by LichKing-2234 — CHANGES_REQUESTED** [review]
需要添加UT对应的Action check

**Comment by guoxianzhe** [comment]
test ci & github comment  done
pls review again @LichKing-2234

File: packages/agora-rtc-react/src/components/CameraVideoTrack.tsx (hunk 1 above)
Comment (guoxianzhe) [reply packages/agora-rtc-react/src/components/CameraVideoTrack.tsx]: already rolled back

**Review by LichKing-2234 — APPROVED** [review]

**Pull request merged**